### ISSUE TITLE
`mode: QueueLength, value: "##"`  does not scale, but the legacy `queueLength: "##"` does.

### ISSUE THREAD

**Issue by PatrickGhosn** [kedacore/keda#2496]
### Report

When we use the new `rabbitmq` trigger configuration with `Keda 2.5`, it does not scale at all. 

```
  triggers:
  - type: rabbitmq
    metadata:
      protocol: amqp
      queueName: testqueue
      mode: QueueLength
      value: "50"
    authenticationRef:
      name: keda-trigger-auth-rabbitmq-conn

```

However when we switch to the deprecated configuration, the scaling works fine.

```
  triggers:
  - type: rabbitmq
    metadata:
      protocol: amqp
      queueName: testqueue
      queueLength: "50"
    authenticationRef:
      name: keda-trigger-auth-rabbitmq-conn
```

### Expected Behavior

The scaling to work with the new configuration.

### Actual Behavior

It only works with the deprecated config.

### Steps to Reproduce the Problem

1. Use the new config
2. Test the scaling in the message queue
3. Switch to the old config
4. Test the scaling in the message queue

### Logs from KEDA operator

```
example
```

### KEDA Version

2.5.0

### Kubernetes Version

1.22

### Platform

Other

### Scaler Details

RabbitMQ

### Anything else?

The issue seems to be encountered by other people as well. I found the solution here:
https://stackoverflow.com/questions/70520404/keda-no-pods-are-scaling
https://stackoverflow.com/questions/67307036/keda-not-scaling-above-1-pod/67319005#67319005

**Comment by RamCohen** [comment]
Can you provide the keda log to see if there are errors loading the rabbitMq scaler configuration ?
I've looked at the code and tests and I don't see an obvious reason why this wouldn't work

**Comment by JorTurFer** [comment]
hi @PatrickGhosn 
Could you provide more info about your problem? We have an e2e test with that parameter (thanks @RamCohen ) and it works. Could you try updating to the latest KEDA version?

**Comment by PatrickGhosn** [comment]
> Can you provide the keda log to see if there are errors loading the rabbitMq scaler configuration ? I've looked at the code and tests and I don't see an obvious reason why this wouldn't work

The problem was that there were no logs being shown. 


> hi @PatrickGhosn Could you provide more info about your problem? We have an [e2e test](https://github.com/kedacore/keda/blob/main/tests/scalers/rabbitmq-queue-amqp.test.ts#L108-L109) with that parameter (thanks @RamCohen ) and it works. Could you try updating to the latest KEDA version?

Updating to the version v2.6.1 (chart version 2.6.2) seems to have resolved the issue. 

Thank you both for your help!

**Comment by tomkerkhove** [comment]
@PatrickGhosn I presume the log issue is not fixed in v2.6.1 or was it as well?

**Comment by PatrickGhosn** [comment]
> @PatrickGhosn I presume the log issue is not fixed in v2.6.1 or was it as well?

No it's showing now. But oddly enough it always shows 1 replica in the log instead of the real number.

`1.6457130054204574e+09  INFO    scaleexecutor   Successfully updated ScaleTarget        {"scaledobject.Name": "test-scaledobject", "scaledObject.Namespace": "test", "scaleTarget.Name": "test1", "Original Replicas Count": 0, "New Replicas Count": 1}`

But all the logs are showing now.

**Comment by tomkerkhove** [comment]
Did it catch up on the replica count or not? :/

Thanks for letting us know
